examples/HttpServer/HelloServerHandler.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace HttpServer
+{
+    using System.Text;
+    using DotNetty.Buffers;
+    using DotNetty.Codecs.Http;
+    using DotNetty.Common.Utilities;
+    using DotNetty.Transport.Channels;
+    using System;
+    using DotNetty.Common;
+
+    sealed class HelloServerHandler : SimpleChannelInboundHandler<IHttpRequest>
+    {
+        static readonly ThreadLocalCache Cache = new ThreadLocalCache();
+
+        sealed class ThreadLocalCache : FastThreadLocal<AsciiString>
+        {
+            protected override AsciiString GetInitialValue()
+            {
+                DateTime dateTime = DateTime.UtcNow;
+                return new AsciiString($"{dateTime.DayOfWeek}, {dateTime:dd MMM yyyy HH:mm:ss z}");
```

Comment:
Also use `DateFormatter.Format(dateTime)` here? For the current one contains culture info.